src/read/cfi.rs

```diff
@@ -808,8 +814,12 @@ impl<R: Reader> _UnwindSectionPrivate<R> for EhFrame<R> {
         0
     }
 
-    fn return_address_register_encoding(_version: u8) -> ReturnAddressRegisterEncoding {
```

Comment:
Since they are both the same now, we should delete this method from the trait.

Reply:
Good point, updated. Would you go even further and also support CIE v4; effectively removing the `compatible_version` method, too?

Reply:
Yeah, given that we already know the doc is wrong, we should do what other consumers do. I haven't recently looked at gdb, but I think that it handles eh_frame and debug_frame identically for version checks.

Reply:
Thanks, updated.

Reply:
Can you delete the `pub enum ReturnAddressRegisterEncoding` definition too, and it's probably worth squashing these commits. Otherwise this looks good.

Reply:
Ah yes, must have missed the warning. Amended.